### PR TITLE
new activity

### PR DESCRIPTION
**Pull request by Zhxrick**

### CODE REVIEW

File: zharick_donado/hojadevida.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+| **Identificación**  | 1043660098 |
+| **Nombre completo** | Zharick Carolina Donado Caliz |
+| **Edad**            | 18 |
+| **Dirección** | Calle 18#17-70 |
+| **Barrio** | soledad centro |
+| **Celular** | 3242828122|
+| **Email** | zharickdonado848@gmail.com |
+| **GitHub** | https://github.com/zhxrick|
+
```
Comment (GoJhon) [inline zharick_donado/hojadevida.md]: @Zhxrick nice logic and code, good job! remember, you need to add the exercises links in "Hoja de vida".

Comment (EmanuelObredor18) [inline zharick_donado/hojadevida.md]: The links to the activities are missing.

**Review by EmanuelObredor18 — APPROVED** [review]
Good job @Zhxrick!!

**Comment by GoJhon** [comment]
@LonelyWalker17 thank you for update the files. @Zhxrick please create a new branch for pull request. This pull request is into main from main.

**Review by GoJhon — CHANGES_REQUESTED** [review]
@LonelyWalker17 thank you for update the files. @Zhxrick please create a new branch for pull request. This pull request is into main from main.

**Pull request closed without merging**